subsquid/src/dbHelper.ts

```diff
@@ -200,6 +201,17 @@ export async function storeHistoricalLockedValue(
   const wsProvider = new WsProvider(chain());
   const api = await ApiPromise.create({ provider: wsProvider });
   const oraclePrices: Record<string, bigint> = {};
+  const assetDecimals: Record<string, number> = {};
+
+  for (const assetId of Object.keys(amountsLocked)) {
+    const asset = await ctx.store.get(Asset, {
+      where: {
+        id: assetId,
+      },
+    });
+
+    assetDecimals[assetId] = asset?.decimals || 12;
```

Comment:
please remove 12 from this line.

Reply:
retain asset to undefined if it is such in this code.

Reply:
@andresmechali we should remove this

Reply:
If I remove this we cannot properly normalize accumulations. Meaning, if for some reason we don't know the number of decimals, how can we sum it with the previous value in a way that makes sense?

Reply:
Yes I understand, this is meant to be exclusively for the FE to use, to provide total locked values without FE needing to get the decimals and do it themselves. As discussed today, ignoring any token that doesn't have decimals. If this is still not convenient I can send them the values of each token and they can add them up as pleased.